src/redux/preferences/reducer.ts

```diff
@@ -44,20 +46,31 @@ export default (state: IPrefState = initialState, action: IAction): IPrefState =
                     }
                 }
             };
+
         case PREF_SET_BLOCKCHAIN_ORDER:
             return {
                 ...state,
                 blockchains: action.data.blockchains
             };
+
         case SET_TEST_NET:
             return {
                 ...state,
                 testNet: !state.testNet
             };
-        case TOGGLE_TOUCH_ID:
-            state = { ...state };
-            state.touchID = !state.touchID;
-            break;
+
+        case TOGGLE_BIOMETRIC_AUTH:
```

Comment:
if you use toggle, why u made a new action for disable? toogle can make enable/disable